__tests__/app.test.js

```diff
@@ -212,6 +212,44 @@ describe('sorting the articles', () => {
 });
 
 
-// Comments should be served with the most recent comments first.
-
+//tests for posting comments 
+
+describe("POST comments", () => {
+  test("adds comment to the article and responds with the posted comment by registered user", () => {
+    //arr 
+    const newComment = {
+      author: 'icellusedkars',
+      body: 'My first comment'
+    }
+    //act 
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual(expect.objectContaining({
+          comment_id: expect.any(Number),
+          body: 'My first comment',
+          author: 'icellusedkars',
+          article_id: 1,
+          created_at: expect.any(String),
+          votes: 0
+        }));
+      });
+  });
+  test("returns 403 status when non-registered user wants to add comment to article", () => {
```

Comment:
lets return a 400 here because the api is going to be used by us in the frontend not the user